lab_solutions/lab02_02_count_elements.py

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2020 Attila Nagy <attila.nagy234@gmail.com>
+#
+# Distributed under terms of the MIT license.
+
+
+def count_elements(sequence):
+    counts = {}
+    for i in sequence:
+        if i not in counts.keys():
```

Comment:
This is more efficient especially for large dictionaries:

```suggestion
        if i not in counts:
```